modules/proc_manage-clairv4.adoc

```diff
@@ -0,0 +1,263 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim <1>
+metadata:
+  name: clairv4-postgres
+  labels:
+    quay-component: clairv4-postgres
+spec:
+  accessModes:
+    - "ReadWriteOnce"
+  resources:
+    requests:
+      storage: "50Gi"
+    volumeName: "clairv4-postgres"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: clairv4-postgres
+  labels:
+    quay-component: clairv4-postgres
+spec:
+  type: ClusterIP
+  ports:
+    - port: 5432
+      protocol: TCP
+      name: postgres
+      targetPort: 5432
+  selector:
+    quay-component: clairv4-postgres
+----
+<1> Confirm that your OpenShift cluster provides the storage features
+noted in the deployment or modify it as necessary.
+
+. Deploy the postgres database as follows:
++
+```
+$ oc create -n $namespace -f ./clairv4-postgres.yaml
+```
+
+. Create the Clair v4 deployment file (for example, `clair-combo.yaml`) and modify it as necessary:
++
+.clair-combo.yaml
+[source,yaml]
+----
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    quay-component: clair-combo
+  name: clair-combo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      quay-component: clair-combo
+  template:
+    metadata:
+      labels:
+        quay-component: clair-combo
+    spec:
+      containers:
+        - image: {productrepo}/clair:{clairnewver}  <1>
+          imagePullPolicy: IfNotPresent
+          name: clair-combo
+          env:
+            - name: CLAIR_CONF
+              value: /clair/config.yaml
+            - name: CLAIR_MODE
+              value: combo
+          ports:
+            - containerPort: 8080
+              name: clair-http
+              protocol: TCP
+            - containerPort: 8089
+              name: clair-intro
+              protocol: TCP
+          volumeMounts:
+            - mountPath: /clair/
+              name: config
+      restartPolicy: Always
+      volumes:
+        - name: config
+          secret:
+            secretName: clairv4-config-secret
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: clairv4 <2>
+  labels:
+    quay-component: clair-combo
+spec:
+  ports:
+    - name: clair-http
+      port: 80
+      protocol: TCP
+      targetPort: 8080
+    - name: clair-introspection
+      port: 8089
+      protocol: TCP
+      targetPort: 8089
+  selector:
+    quay-component: clair-combo
+  type: ClusterIP
+----
+<1> Change image to latest clair image name and version.
+<2> With the Service set to clairv4, the scanner endpoint for Clair v4
+is entered later into the {productname} config.yaml in the
+SECURITY_SCANNER_V4_ENDPOINT as http://clairv4.
+
+. Create the Clair v4 deployment as follows:
++
+```
+$ oc create -n $namespace -f ./clair-combo.yaml
+```
+
+. Modify the `config.yaml` file for your {productname} deployment to add the following
+two entries at the end:
++
+[source,yaml]
+----
+SECURITY_SCANNER_V4_ENDPOINT: http://clairv4 <1>
+SECURITY_SCANNER_V4_NAMESPACE_WHITELIST: <2>
+  - "clairv4-org"
+  - "foo-org"
+----
```

Comment:
Suggest to add `FEATURE_SECURITY_SCANNER: true`

Reply:
Done.